bundles/best_practices.rst

```diff
@@ -166,6 +166,73 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of Symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+            - $HOME/symfony-bridge/.phpunit
+
+    env:
+        global:
+            - PHPUNIT_FLAGS="-v"
+            - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported Symfony version with the latest PHP version
+            - php: 7.2
```

Comment:
i think it would be better to do the minium install with the oldest supported PHP version. that way we make sure to end up with the worst possible legacy combination of packages. some of our dependencies might not allow PHP 7.2 in their oldest versions...

Reply:
But there is more chances that an old version of a library fail with a cutting edge PHP version than with an old (contemporary) version.

Reply:
hm, i see your point. imho tests should mainly test this library and its interactions work. when you allow php 5.6 the problem is even larger, the --prefer-lowest might not install the oldest thing you allow in composer.json but only a newer major version. should we add both, with the explanation that the newest php gives a plausibility check whether our dependencies work with the newest version of php?